packages/dev/test/RelayHub.test.ts

```diff
@@ -82,20 +82,20 @@ contract('RelayHub', function ([_, relayOwner, relayManager, relayWorker, sender
     async function testDeposit (sender: string, paymaster: string, amount: BN): Promise<void> {
       const senderBalanceTracker = await balance.tracker(sender)
       const relayHubBalanceTracker = await balance.tracker(relayHub)
-
-      const { logs } = await relayHubInstance.depositFor(paymaster, {
+      const gasPrice = new BN(1e9)
+      const res = await relayHubInstance.depositFor(paymaster, {
         from: sender,
         value: amount,
-        gasPrice: 0
+        gasPrice
       })
-      expectEvent.inLogs(logs, 'Deposited', {
+      expectEvent.inLogs(res.logs, 'Deposited', {
         paymaster,
         from: sender,
         amount
       })
-
+      const txCost = (new BN(res.receipt.gasUsed)).mul(gasPrice)
       expect(await relayHubInstance.balanceOf(paymaster)).to.be.bignumber.equal(amount)
-      expect(await senderBalanceTracker.delta()).to.be.bignumber.equal(amount.neg())
+      expect(await senderBalanceTracker.delta()).to.be.bignumber.equal(amount.neg().sub(txCost))
```

Comment:
.add().neg() maybe? took me a second to understand wtf is going on here

Reply:
Same difference :laughing: